assets/javascripts/application.js

```diff
@@ -60,3 +61,54 @@ $( document ).ready( function(){
 
   post_expansion_setup();
 });
+
+var rootSelector = ".space_for_appsight"
```

Comment:
these all need to be global functions so that both pod inline search results and pod pages can access it